apis/nucleus/src/__tests__/viz.test.js

```diff
@@ -232,6 +240,26 @@ describe('viz', () => {
     });
   });
 
+  describe('toggleDataView', () => {
+    test('should toggle data view and back', async () => {
```

Comment:
This would pass if `api.toggleDataView()` did not actually convert it to type `sn-table`. So consider adding a check for that as well.